package.json

```diff
@@ -64,14 +64,16 @@
         "spectrum-tokens": "node ./scripts/spectrum-tokens.js",
         "spectrum-vars": "node ./scripts/spectrum-vars.js",
         "start": "yarn storybook",
-        "storybook": "yarn build:ts && yarn build:css && run-p storybook:start watch:css build:watch",
+        "storybook": "yarn build && yarn storybook:quick",
         "storybook:build": "build-storybook -o projects/documentation/dist/storybook",
-        "storybook:start": "web-dev-server --config wds-storybook.config.js",
+        "storybook:quick": "run-p watch:css build:watch storybook:run",
+        "storybook:run": "wds --config wds-storybook.config.js",
         "test": "yarn test:focus unit",
         "test:bench": "node test/benchmark/cli.cjs",
-        "test:build": "tsc --build test/tsconfig-plugins.json && tsc --build test/tsconfig-test.json",
+        "test:build": "yarn build",
```

Comment:
Interesting - can you outline this change for me? Test is running the generic build - does that now include building test assets including the plugins?

Reply:
Good call. I was namely pruning commands back as I renamed them as things. Will clean this up.